src/utils/helpers.ts

```
@@ -0,0 +1,21 @@
+            currency: 'USD',
+            maximumFractionDigits: 2,
+        }).format(amount);
+    }
+
+    const compact = new Intl.NumberFormat('en-US', {
+        notation: 'compact',
+        compactDisplay: 'short',
+        maximumFractionDigits: 2,
+    }).format(amount);
+
+    return (amount < 0 ? '-' : '') + '$' + compact.replace(/^-/, '');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)